### PR TITLE
1.18.6 Release Notes

### PR DESCRIPTION
**Pull request by pmerrison**
## Description

Release notes for 1.18.6.  Linter will fail until https://github.com/istio/istio.io/pull/14302 is merged, as it refers to links that don't exist yet

## Reviewers

- [ ] Ambient
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [ ] Localization/Translation

### CODE REVIEW

File: content/en/news/releases/1.18.x/announcing-1.18.6/index.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+title: Announcing Istio 1.18.6
+linktitle: 1.18.6
+subtitle: Patch Release
+description: Istio 1.18.6 patch release.
+publishdate: 2023-12-11
+release: 1.18.6
+---
+
+This release implements the security updates described our Dec 11th post, [`ISTIO-SECURITY-2023-005`](/news/security/istio-security-2023-005) along with bug fixes to improve robustness.
```
Comment (ericvn) [inline content/en/news/releases/1.18.x/announcing-1.18.6/index.md:10]:
```suggestion
This release implements the security updates described in our Dec 11th post, [`ISTIO-SECURITY-2023-005`](/news/security/istio-security-2023-005) along with bug fixes to improve robustness.
```

File: content/en/news/releases/1.18.x/announcing-1.18.6/index.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+This release note describes what’s different between Istio 1.18.5 and 1.18.6. This is the last planned release for Istio 1.18, for more details see our Nov 29th [end of support announcement](/news/support/announcing-1.18-eol/)
```
Comment (ericvn) [inline content/en/news/releases/1.18.x/announcing-1.18.6/index.md:12]:
```suggestion
This release note describes what’s different between Istio 1.18.5 and 1.18.6. This is the last planned release for Istio 1.18, for more details see our Nov 29th [end of support announcement](/news/support/announcing-1.18-eol/).
```

File: content/en/news/releases/1.18.x/announcing-1.18.6/index.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+- **Improved** `iptables` locking. The new implementation uses `iptables` builtin lock waiting when needed, and disables locking entirely when not needed.
+
+- **Fixed** DNS Proxy resolution for wildcard ServiceEntry with the search domain suffix for glibc based containers.
+  ([Issue #47290](https://github.com/istio/istio/issues/47290)),([Issue #47264](https://github.com/istio/istio/issues/47264)),([Issue #31250](https://github.com/istio/istio/issues/31250)),([Issue #33360](https://github.com/istio/istio/issues/33360)),([Issue #30531](https://github.com/istio/istio/issues/30531)),([Issue #38484](https://github.com/istio/istio/issues/38484))
+
+- **Fixed** an issue where using a Sidecar resource using `IstioIngressListener.defaultEndpoint` cannot use [::1]:PORT if the default IP addressing is not IPv6.
+  ([Issue #47412](https://github.com/istio/istio/issues/47412))
+
+- **Fixed** an issue where `istioctl proxy-config` fails to process a config dump from file if EDS endpoints were not provided.
```
Comment (ericvn) [inline content/en/news/releases/1.18.x/announcing-1.18.6/index.md:26]: Slightly confusing. Not sure if this is correct or not:
```suggestion
- **Fixed** an issue where `istioctl proxy-config` fails to process a config dump from a file if EDS endpoints were not provided.
```

File: content/en/news/releases/1.18.x/announcing-1.18.6/index.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  ([Issue #47893](https://github.com/istio/istio/issues/47893))
+
+- **Fixed** a memory leak when `WorkloadEntries` change their IP address.
+  ([Issue #47893](https://github.com/istio/istio/issues/47893))
+
+- **Fixed** a memory leak when a `ServiceEntry` is removed. 
+  ([Issue #47893](https://github.com/istio/istio/issues/47893))
+
+# Security update
+- Changes to Istio CNI Permissions as described in [`ISTIO-SECURITY-2023-005`](/news/security/istio-security-2023-005)
```
Comment (ericvn) [inline content/en/news/releases/1.18.x/announcing-1.18.6/index.md:54]:
```suggestion
- Changes to Istio CNI Permissions as described in [`ISTIO-SECURITY-2023-005`](/news/security/istio-security-2023-005).
```

**Review by dhawton — APPROVED** [review]
Other than date push as lint won't pass until #14302 is merged which is likely tomorrow since it's past midnight for you, LGTM.

File: content/en/news/releases/1.18.x/announcing-1.18.6/index.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
+---
+title: Announcing Istio 1.18.6
+linktitle: 1.18.6
+subtitle: Patch Release
+description: Istio 1.18.6 patch release.
+publishdate: 2023-12-12
+release: 1.18.6
+---
+
+This release implements the security updates described in our Dec 11th post, [`ISTIO-SECURITY-2023-005`](/news/security/istio-security-2023-005) along with bug fixes to improve robustness.
```
Comment (ericvn) [inline content/en/news/releases/1.18.x/announcing-1.18.6/index.md:10]:
```suggestion
This release implements the security updates described in our Dec 12th post, [`ISTIO-SECURITY-2023-005`](/news/security/istio-security-2023-005) along with bug fixes to improve robustness.
```

**Review by ericvn — APPROVED** [review]

File: content/en/news/releases/1.18.x/announcing-1.18.6/index.md (hunk 5 above)
Comment (ericvn) [inline content/en/news/releases/1.18.x/announcing-1.18.6/index.md:10]:
```suggestion
This release implements the security updates described in our Dec 12th post, [`ISTIO-SECURITY-2023-005`](/news/security/istio-security-2023-005) along with bug fixes to improve robustness.
```

**Comment by ericvn** [comment]
Can we remove the https://github.com/istio/istio.io/pull/14303 commit that was added?

**Pull request closed without merging**